### PR TITLE
Ru translate for auth

### PR DESCRIPTION
**Pull request by rndtrash**

### CODE REVIEW

File: ru/auth.toml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+# After a successful sign up they land here.
+# They still have to authorize their account.
+[auth.join.almost]
+	page_title = "Почти готово!"
+	heading = "@:auth.join.almost.page_title"
+	body_html = """
+	<p>Перед тем как войти в аккаунт Game Jolt.</p>
+	<p>Мы отправили вам инструкцию по активации.</p>
+	"""
+	spam = "If you don't see an email within 10 minutes, please check your spam folder. It might have gobbled it up."
```
Comment (hworld) [inline ru/auth.toml:60]: This one needs a translation.

File: ru/auth.toml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	urgency = "Note that you have a week to authorize your account. After that, it will be deleted and you'll have to sign up again."
```
Comment (hworld) [inline ru/auth.toml:61]: This one needs a translation.

File: ru/auth.toml [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+	heading = "@:auth.forgot.sent.page_title"
+	body_html = "<p>Мы отправили вам инструкцию по электронной почте.</p>"
+	spam = "Если вы не увидите сообщение через 10 минут, проверьте папку со спамом. Возможно оно находится там!"
+
+# The URL for this gets sent in an email when they fill out the forgot login credentials page.
+[auth.reset_password]
+	page_title = "Сбросить пароль"
+	heading = "@:auth.reset_password.page_title"
+
+	password_help = "If you'd like to reset your password, enter a new one below."
```
Comment (hworld) [inline ru/auth.toml:107]: This one needs a translation.

File: ru/auth.toml [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+	created_growl = "Ваш профиль создан. Теперь вы можете отправиться в плаванье!"
+	created_growl_title = "Профиль создан"
+
+# /login/finalize-twitter
+# They get directed here when they sign up with a new twitter account and we require their email address.
+[auth.linked_account.twitter.finalize]
+	page_title = "Введите Email"
+	heading = "@:auth.linked_account.twitter.finalize.page_title"
+
+	email_help = "To finish logging in with Twitter, please enter a valid email address."
```
Comment (hworld) [inline ru/auth.toml:163]: This one needs a translation.

**Comment by hworld** [comment]
Hey there! Just a few missing.

**Comment by hworld** [comment]
I'm going to merge this in anyway so that I can pull it into the new translation management system. From there you'll be able to make updates much easier.

**Comment by rndtrash** [comment]
Well I'll fix it

**Pull request merged**